packages/tools/src/tools/annotation/LivewireContourTool.ts

```diff
@@ -72,6 +67,43 @@ class LivewireContourTool extends ContourSegmentationBaseTool {
       supportedInteractionTypes: ['Mouse', 'Touch'],
       configuration: {
         preventHandleOutsideImage: false,
+        /**
+         * Configuring this to a value larger than 0 will snap handles to nearby
+         * livewire points, within the given rectangle surrounding the clicked point.
+         */
+        snapHandleNearby: 1,
+
+        /**
+         * Interpolation is only available for segmentation versions of these
+         * tools.  To use it on the segmentation tools, set enabled to true,
+         * and create two livewire contours in the same segment index, separated
+         * by at least one slice.
+         */
+        interpolation: {
+          enabled: false,
+
+          /**
+           * Set the nearestEdge to snap interpolated handles to an edge within
+           * the given number of pixels.  Setting to 0 disables snap to pixel
+           * for interpolation.
+           */
+          nearestEdge: 0,
+          /**
+           * Sets a CSS color to use for showing the interpolation contour overlaying
+           * the livewire contour for auto generated contours.
+           */
+          interpolationColor: <string>null,
```

Comment:
I didn't expect to find this here. Ideally, it should be a property in the ToolStyle, rather than in the tool instance. Similar to other appearance configs